ChangeLog

```diff
@@ -1,3 +1,9 @@
+2015-07-27  Sherine Awad  <drmahmoud@ucdavis.edu>
+
+   * Add a line break in multi-rename.py 
```

Comment:
Please fix this format to look like the other ChangeLog entries: for each line, '*', then filename, followed by colon, followed by description.